Formula/clojure-lsp.rb

```diff
@@ -15,12 +16,17 @@ class ClojureLsp < Formula
   end
 
   depends_on "leiningen" => :build
+  depends_on "openjdk@11"
 
   def install
     system "lein", "uberjar"
     jar = Dir["target/clojure-lsp-*-standalone.jar"][0]
     libexec.install jar
-    bin.write_jar_script libexec/File.basename(jar), "clojure-lsp"
+    (bin/"clojure-lsp").write <<~EOS
+      #!/bin/bash
+      export JAVA_HOME="${JAVA_HOME:-#{Formula["openjdk@11"].opt_prefix}}"
+      exec "${JAVA_HOME}/bin/java" -jar "#{libexec}/#{File.basename(jar)}" "$@"
+    EOS
```

Comment:
Not really specific to this formula, but would it be worth having `write_jar_script` (or a new method with a similar purpose) work with brew OpenJDK?

Reply:
Sounds good

Reply:
Yes, this would be great 👍